site/interfaces/index.ts

```diff
@@ -69,3 +69,38 @@ export interface AddConsequenceProps {
     modeOfTransport?: TransportMode;
     consequenceType?: ConsequenceType;
 }
+
+export interface Consequence {
+    "mode-of-transport": TransportMode;
+    "consequence-type": ConsequenceType;
+    service?: { id: string; name: string }[];
```

Comment:
Should this be services as its an array?

Reply:
Also wonder if it should be services-affected, as we have stops-affected

Reply:
renamed